### ISSUE TITLE
BPMN.Sharp library is not open source ?

### ISSUE THREAD

**Issue by guilmori** [bzinchenko/bpmnview#1]
BPMN.Sharp library is not open source ?

**Comment by bzinchenko** [comment]
BPMN.Sharp library is available as Nuget package under MIT license. It is not open source though.

**Comment by borseno** [comment]
Yes but it's only available for .NET framework so I have to get the sources and go through them to migrate them to .NET Core?

**Comment by borseno** [comment]
Is it ok if I decompile the DLL and migrate it to .NET Core?

**Comment by BadgerTheAdger** [comment]
![image](https://user-images.githubusercontent.com/33373431/91732457-ed866200-ebb0-11ea-9205-3ef0bb15882c.png)
Oh shit.

**Comment by borseno** [comment]
![image](https://user-images.githubusercontent.com/33373431/91732638-29b9c280-ebb1-11ea-9ad3-5a7653345e92.png)
Oh shit.

Sir, I tried doing that but I ran into this.

Then I renamed all unicodes to normal texts.
![image](https://user-images.githubusercontent.com/33373431/91736220-05141980-ebb6-11ea-8041-5e74c21b85b0.png)

Still doesn't work. This nuget stuff went even further - it made methods and fields have the same names in the same class
